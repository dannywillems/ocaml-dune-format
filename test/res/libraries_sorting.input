(library
 (name mylib)
 (libraries stdio lwt base unix)
 (modules mylib))

(executable
 (name main)
 (libraries yojson base stdio)
 (modules main))

(test
 (name test_runner)
 (libraries ounit2 mylib base)
 (modules test_runner))