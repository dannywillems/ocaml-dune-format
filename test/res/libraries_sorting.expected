(library
 (name mylib)
 (libraries base lwt stdio unix)
 (modules mylib))

(executable
 (name main)
 (libraries base stdio yojson)
 (modules main))

(test
 (name test_runner)
 (libraries base mylib ounit2)
 (modules test_runner))